app/src/main/java/com/example/android_challenge_02/QuoteList.kt

```diff
@@ -24,5 +29,12 @@ class QuoteList : AppCompatActivity() {
             val intent = Intent(this, MainActivity::class.java)
             startActivity(intent)
         }
+
```

Comment:
Levando em considerações os padrões de código limpo é bom fazer o código pensando em deixa-lo o mais enxuto possível pra que outras pessoas possam entende-lo também.
Recomendo pra diminuir o codigo setar o recycler view numa tacada só usando apply.
`binding.recyclerView.apply{
    adapter =  PrototypeAdapter()
    layoutManager = LinearLayoutManager(this)
}`
dessa forma seu código ficara muito mais limpo.